doc/network-forwards.md

```diff
@@ -1,59 +1,96 @@
 # How to configure network forwards
 
-The following network types support forwards. See each network type section for more details.
+```{note}
+Network forwards are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
- - [bridge](#network-bridge)
- - [ovn](#network-ovn)
+Network forwards allow an external IP address (or specific ports on it) to be forwarded to an internal IP address (or specific ports on it) in the network that the forward belongs to.
 
- Network forwards allow an external IP address (or specific ports on it) to be forwarded to an internal IP address (or specific ports on it) in the network that the forward belongs to.
+This feature can be useful if you have only one public IP address, but you don't want to statically assign this address to one instance only.
```

Comment:
I'm not sure about this line. Firstly, the term "public" is not correct here, just because the listen IP of the forward is external to the LXD network its being added to does not make it automatically "public" (in the global sense of the word). I would use the word "external" as the IP is external to the LXD managed network, which also covers truly public IPs, but doesn't suggest this feature is only for use with public IPs.

Also the word "assign" is problematic to me also, as it suggests that there is an ability to use the IP actually inside the instance, which may or may not be possible depending on the external network setup. Instead I would suggest a statement like:

"This feature can be useful when you have limited external IP addresses and want to share a single external IP address between multiple instances".